pydriller/domain/commit.py

```diff
@@ -173,6 +174,41 @@ def token_count(self) -> int:
         self._calculate_metrics()
         return self._token_count
 
+    @property
+    def methods_modified(self) -> List[Method]:
+        """
+        Return the list of methods modified in the file. Every method
+        contains various information like complexity, loc, name,
+        number of parameters, etc.
+
+        :return: list of methods modified
+        """
+        self._identified_methods_modified()
+        return self._function_list_modified
+
+    def _identified_methods_modified(self):
+        if self.source_code and self._nloc is None:
+            l = lizard.analyze_file.analyze_source_code(self.filename, self.source_code)
+
+            self._nloc = l.nloc
+            self._complexity = l.CCN
+            self._token_count = l.token_count
+
+            lines = self.diff.split('\n')
```

Comment:
I don't think this is correct, because you parse the diff to obtain the numbers between @@. Like @@ 1,6 1,7 @@. However, you do not know specifically what line number changed (e.g., was it line number 2, or number 5?). 

I think what you should do is something similar to the function **parse_diff** ([Link](https://github.com/ishepard/pydriller/blob/aee50f80543ec27bc5d4d490620602bc87fa6794/pydriller/git_repository.py#L182)). You are interested in the specific line number that have been changed.